.changeset/olive-laws-thank.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-azure-devops-backend': minor
```

Comment:
Are the all `patch` releases because the versions are `<1.x`?

Reply:
I think they are `minor` changes as they would cause breaking changes for public consumers.

Reply:
Ah ok - sorry missed the `memberId`s change in the `api-report.md`